test/Microsoft.AspNet.Mvc.TagHelpers.Test/LinkTagHelperTest.cs

```diff
@@ -451,8 +440,9 @@ public static TheoryData DoesNotRunWhenARequiredAttributeIsMissing_Data
             helper.Process(context, output);
 
             // Assert
-            Assert.NotNull(output.TagName);
             Assert.False(output.IsContentModified);
+            Assert.Empty(output.Attributes);
+            Assert.Empty(output.PostElement.GetContent());
```

Comment:
`Assert.True(output.PostElement.IsEmpty)` instead throughout your changes. Only need to call `GetContent()` when looking for specific content.